Makefile

```diff
@@ -26,7 +32,11 @@ GPGME_ENV = CGO_CFLAGS="$(shell gpgme-config --cflags 2>/dev/null)" CGO_LDFLAGS=
 all: tools test validate .gitvalidation
 
 build: vendor
-	@$(GPGME_ENV) go build $(BUILDFLAGS) $(PACKAGES)
+	$(GPGME_ENV) go build $(BUILDFLAGS) $(PACKAGES)
```

Comment:
Is echoing the hard-to-parse list of packages that useful?

Reply:
debugging. I'll add it back.